cli/dcoscli/log.py

```diff
@@ -6,10 +6,14 @@
 import time
 
 import six
+from six.moves import urllib
```

Comment:
we no longer support python2.x so we don't need to use six if you don't want to :)